hackathon_site/dashboard/frontend/src/components/cart/CartErrorBox/index.tsx

```diff
@@ -19,6 +27,18 @@ const CartErrorBox = () => {
                         title="Unable to submit your order because:"
                     />
                 )}
+                {(teamSizeTooLarge || teamSizeTooSmall) && (
+                    <Alert severity="warning" style={{ margin: "15px 0px" }}>
+                        <AlertTitle>{`Team Size Too ${errorTitle}!`}</AlertTitle>
+                        {`There are too ${errorMessage} people on your team to place an
+                        order. We only allow teams between ${minTeamSize} to ${maxTeamSize} 
+                        to checkout items. To join or leave a team please
+                        go to your `}
+                        <a href="/">Dashboard </a>
```

Comment:
<img width="561" alt="image" src="https://user-images.githubusercontent.com/20618460/172070985-7b1e2c26-d8ef-4087-8fc1-e4a2d28147c6.png">

here, "Dashboard" doesn't look very noticeable, could you use MUI's link component instead https://v4.mui.com/components/links/ to give it an underline